rollup.config.addon.js

```diff
@@ -20,7 +20,7 @@ export default (cliArgs) => {
           "'https://firefox.settings.services.mozilla.com/v1/buckets/main/collections/pioneer-study-addons-v1/records'",
         __ION_WEBSITE_URL__: cliArgs['config-website'] ?
           `'${cliArgs['config-website']}'` :
-          "'https://mozilla-rally.github.io'",
+          "'https://rally-stage.bespoke.nonprod.dataops.mozgcp.net'",
```

Comment:
Can you kindly add a changelog entry for this?

Reply:
@Dexterp37 Oh I just did so, but since the PR must be in the changelog don't I need to open the PR first ? :)

Reply:
Yeah, that's an unfortunately small caveat of our release process! I'm open to suggestions on this :)